modules/kn-service-describe.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative_serving/serverless-applications.adoc
+// * serverless/cli_reference/kn-services-ref.adoc
+
+[id="kn-service-describe_{context}"]
+= Describing serverless applications
+
+You can describe a Knative service by using the `kn` CLI `kn service describe` command.
+
+.Example commands
+
+* Describe a service:
++
+[source,terminal]
+----
+$ kn service describe <service-name>
+----
++
+.Example output
++
+[source,terminal]
+----
+Name:         example-service
+Namespace:    knative-serving
+Annotations:  serving.knative.openshift.io/ownerName=knative-serving, serving.knative.openshift.i ...
+Age:          2h
+URL:          http://example-service-knative-serving.apps.ocp1.redhat.com
+Revisions:
+  100%  @latest (example-service-00002) [2] (2h)
+        Image:  registry.redhat.io/openshift-serverless-1/example-service-artifacts-rhel8@sha256:e75ffcb0d051f9abb945eaa91ae281858387f92f65c884a49b8801560cbf4965 (at e75ffc)
+Conditions:
+  OK TYPE                   AGE REASON
+  ++ Ready                   2h
+  ++ ConfigurationsReady     2h
+  ++ RoutesReady             2h
+----
+
+* Describe a service in YAML format:
++
+[source,terminal]
+----
+$ kn service describe <service-name> -o yaml
+----
+
+* Print the service URL only:
++
+[source,terminal]
+----
+$ kn service describe <service-name> -o url
+----
```

Comment:
We probably also should mention `-v` (`--verbose`) becaust this will add some extra information that might be of interest. For example, the annotation list is not truncated as in the example above, but listed line-by-line. Let me know if you need a sample output for this.

Reply:
Yeah sample output for this would be good then I'm happy to add the command with `-v` as another option?

Reply:
yep, it's just `kn service describe -v` (or `kn service describe --verbose`). Unfortunately, I don't have a cluster running right now (nor can I reproduce the example here), but can deliver the output tomorrow.

Reply:
@rhuss do you still want to include this? Otherwise we can follow up another time with adding the verbose option if you don't have the output for now.

Reply:
I will send you the required output tomorrow. `NoClusterException` right now :)

Reply:
Sorry for not holding my promises 😞  too much around here, one call hunts the other. Let's merge it now and tune the describe the output later. There are other things in the example output that would need fixing, too (e..g a user does not install her services in the `knative-serving` namespaces as shown in the sample. 

Also I would need to have the same service for creating a corresponding sample output with `-v`

Reply:
No problem, thanks Roland!